### DISCUSSION TITLE
Q: 绑定域名后提示重定向次数过多 || Q: After binding the domain name, it prompts that there are too many redirects

### DISCUSSION THREAD

**Discussion by haloged** [walinejs/waline#1986] [Q&A] [answered]
### Describe the bug | 问题描述

Vercel绑定域名后登录提示提示重定向次数过多（pl.haloged.eu.org）

### Website URL | 问题网站

https://haloged.eu.org/2023/05/14/untitled-1684066642418/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

After Vercel binds a domain name, the login prompt prompts too many redirects (pl.haloged.eu.org)

### Website URL | Problem Website

https://haloged.eu.org/2023/05/14/untitled-1684066642418/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment]
是不是 cf 里面做了啥重定向的设置，可以自己检查下。Waline 没有这个策略

---
Is there any redirection setting in cf, you can check it yourself. Waline doesn't have this policy

**Reply by haloged** [reply]
没有啊
![image](https://github.com/walinejs/waline/assets/101408211/7301988c-c82e-438e-8f43-ec91b4766978)

**Reply by lizheming** [reply, marked as answer]
https://vercel.com/guides/resolve-err-too-many-redirects-when-using-cloudflare-proxy-with-vercel 看下这个
